lib/tbot/cli/start_shared.go

```diff
@@ -123,14 +124,16 @@ type sharedStartArgs struct {
 
 // newSharedStartArgs initializes shared arguments on the given parent command.
 func newSharedStartArgs(cmd *kingpin.CmdClause) *sharedStartArgs {
+	cmd.Interspersed(true)
```

Comment:
(implementation note) This impacts all commands, but I noticed (at least) arguments could not precede flags.

Reply:
No longer relevant as we're using a flag now.